openscenario/map_fragment/src/lane.cpp

```diff
@@ -53,15 +54,8 @@ try {
     return std::filesystem::path(node.get_parameter("output_directory").as_string());
   }();
 
-  auto lanelets = lanelet::Lanelets();
-
-  lanelets.push_back(makeLanelet(width, length, curvature, resolution));
-
-  for (auto i = 1; i < number_of_lanes; ++i) {
-    lanelets.push_back(makeLaneletRight(lanelets.back(), resolution));
-  }
-
-  const auto map = lanelet::utils::createMap(lanelets);
+  const auto segment = makeCurvedRoadSegment(curvature, length, number_of_lanes, width);
```

Comment:
Yes, having positive curvature/angle mean a left-turning curve is also the more intuitive option for me. This is reflected in [the documentation](https://github.com/tier4/scenario_simulator_v2/pull/1108/files#diff-7ceb6a20f0efb5b296f7f37da8b442048d21076e185f1a9276f4508daa5dce9fR114) for the `Arc` class. Perhaps a mistake slipped in; I will look into that.

Reply:
Left and right boundaries of lanelets were swapped. Because of that, the direction was reversed and what should be a left turn appeared as a right turn. Fixed in [9bb3669](https://github.com/tier4/scenario_simulator_v2/pull/1108/commits/9bb3669e3aa12de5ef9ddb1122ce145e3d727d51) and [b2abcf2](https://github.com/tier4/scenario_simulator_v2/pull/1108/commits/b2abcf20d99614ef1d4f672cc7e53afe4ed77486).